pandas/tests/frame/test_to_csv.py

```diff
@@ -43,24 +49,30 @@ def test_to_csv_from_csv1(self):
 
             # test roundtrip
             self.tsframe.to_csv(path)
-            recons = DataFrame.from_csv(path)
-
+            recons = self.read_csv(path)
             assert_frame_equal(self.tsframe, recons)
 
+            with tm.assert_produces_warning(FutureWarning,
+                                            check_stacklevel=False):
```

Comment:
the `check_stacklevel=False` should not be needed

Reply:
True, fixed.